README.md

```diff
@@ -11,6 +11,8 @@ We included our own wrapper for the environment as well to support the modificat
 within an openAI gym interface, so that new and more complex tasks can be developed efficiently to 
 train and test the agent.
 
+We will release a blog soon which will go into more detail about this repo and the first results of algorithms trained on these tasks.
```

Comment:
```suggestion
We will release a blog soon which will go into more detail about this repo and how to exploit all of its features on these tasks.
```

Reply:
Or sth in those lines. Algorithms comes later

Reply:
and how to use all of the features of our wrapper.